hack/lib/golang.sh

```diff
@@ -240,6 +242,10 @@ kube::golang::set_platform_envs() {
       export CGO_ENABLED=1
       export CC=powerpc64le-linux-gnu-gcc
     fi
```

Comment:
oh, this `fi` needs to be removed.

might be worth turning this massive `if/elif/elif/elif/...` block into a `case` block.